addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -322,9 +322,12 @@ def _rtc_join_call(self, store: Store = None, check_rtc_session_ids=None, camera
                     "serverInfo": self._get_rtc_server_info(rtc_session, ice_servers),
                 },
             )
-        if len(self.channel_id.rtc_session_ids) == 1 and self.channel_id.channel_type != "channel":
-            self.channel_id.message_post(body=_("%s started a live conference", self.partner_id.name or self.guest_id.name), message_type='notification')
-            self._rtc_invite_members()
+        if len(self.channel_id.rtc_session_ids) == 1:
+            body = Markup('<div data-oe-type="call" class="o_mail_notification"></div>')
+            message = self.channel_id.message_post(body=body, message_type="notification")
+            self.channel_id.last_call_message_id = message
+            if self.channel_id.channel_type != "channel":
+                self._rtc_invite_members()
```

Comment:
I don't know why the notification was not shown in channel, since using call there works and sometimes preferred.
The notification becomes even more useful with the duration time being logged.

The invite members should indeed not be triggered in channels, but other than that the notification is good in all conversations. @seb-odoo @aju-odoo any objection about this change?